index.js

```diff
@@ -129,7 +173,14 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  movieArray = []
+  for (let i = 0; i < movies.length; i ++){
+    if (movies[i].released <= year){
+      movieArray.push(movies[i].released)
+    }return movieArray
+  }
```

Comment:
Our indentation gets a little sloppy here.  Also take a look at where in the function we are returning - is this where we want to return from?